tests/integration/mysql_test.go

```diff
@@ -0,0 +1,218 @@
+// +build integration
```

Comment:
This file removes many unuseful tests from the original. E.g. testing that, if you set a wrong password, MySQL returns an error.

Reply:
Blame of me... I didn't read this useful message. Thanks for commenting this!